lib/rules/one-var.js

```diff
@@ -12,25 +13,58 @@
 
 module.exports = function(context) {
 
+    var MODE = context.options[0];
```

Comment:
Make sure to provide a default value of `"always"` if `context.options[0]` is not provided.